### PR TITLE
adding simple function to check it site is building

### PR DESCRIPTION
**Pull request by JulianGindi**
Making it so we can't build again if the same site is already building...need a way to inform the user of this.

### CODE REVIEW

File: franklin/builder/models.py [hunk 1]
```diff
@@ -224,15 +224,21 @@ def build(self):
                 "environment_id": self.id
                 }
 
-            response = make_rest_post_call(url, headers, body)
-            if not status.is_success(response.status_code):
-                logger.error("Negative response from Builder: %s",
-                             response.status_code)
-                self.status = self.FAILED
+            if self.can_build():
```
Comment (oudeismetis) [inline franklin/builder/models.py]: why not do this at line 208?

Comment (JulianGindi) [inline franklin/builder/models.py]: probably a better place to put it

**Comment by oudeismetis** [comment]
Otherwise seems very reasonable :+1:

File: franklin/builder/models.py (hunk 1 above)
Comment (oudeismetis) [inline franklin/builder/models.py]: even better, add this to line 208 and then move `if self.current_deploy` inside of the `can_build()` function.

**Comment by oudeismetis** [comment]
awesome. Pull it :+1:

**Pull request closed without merging**